pyuvdata/uvbeam.py

```diff
@@ -150,7 +150,9 @@ def __init__(self):
                 'Peak normalized means that for each frequency the data_array'
                 'is separately normalized such that the peak is 1 (so the beam '
                 'is dimensionless) and all frequency dependence is moved to the '
```

Comment:
It is not true that all frequency dependence is moved to the bandpass. The beam is still frequency dependent in general -- i.e. is not a separable function of space and frequency -- for any normalization.